ast/misc.go

```diff
@@ -69,6 +69,7 @@ const (
 	ExplainFormatJSON    = "json"
 	ExplainFormatHint    = "hint"
 	ExplainFormatVerbose = "verbose"
+	ExplainFormatTest    = "test"
```

Comment:
Does `test` seems informal and too casual? How about `brief` or some names more specified like `without-id`?